src/components/books.js

```diff
@@ -1,32 +1,42 @@
+import React from 'react';
 import './books.css';
+import { useDispatch } from 'react-redux';
+import { removeBook, Book } from '../redux/books/books';
 
-const Books = () => (
-  <div className="Lesson-Panel">
-    <div className="book-info">
-      <span className="School-of">Action</span>
-      <h2 className="Title">The Hunger Games</h2>
-      <span className="author">Suzanne-Collins</span>
-      <div className="options">
-        <span className="actions">Comments</span>
-        <span className="actions">Remove</span>
-        <span className="actions">Edit</span>
+const Books = () => {
+  const dispatch = useDispatch();
+  const removeBookFromStore = () => {
+    dispatch(removeBook(Book));
+  };
+
+  return (
+    <div className="Lesson-Panel">
+      <div className="book-info">
+        <span className="School-of">Action</span>
+        <h2 className="Title">The Hunger Games</h2>
+        <span className="author">Suzanne-Collins</span>
+        <div className="options">
+          <span className="actions">Comments</span>
+          <button type="button" className="actions" onClick={removeBookFromStore}>Remove</button>
+          <span type="button" className="actions">Edit</span>
+        </div>
```

Comment:
- [ ] There should be no default Books, all books should be added and removed using the redux store. Kindly modify this.